src/core/public/doc_links/doc_links_service.ts

```diff
@@ -198,6 +198,9 @@ export class DocLinksService {
           transportSettings: `${ELASTICSEARCH_DOCS}modules-network.html#common-network-settings`,
           typesRemoval: `${ELASTICSEARCH_DOCS}removal-of-types.html`,
           deprecationLogging: `${ELASTICSEARCH_DOCS}logging.html#deprecation-logging`,
+          setupUpgrade: `${ELASTICSEARCH_DOCS}setup-upgrade.html`,
+          whatsNew: `${ELASTICSEARCH_DOCS}release-highlights.html`,
```

Comment:
Can we rename this `releaseHighlights` to correspond with the URL, rather than the page heading? I think it's a little more precise in describing the role of these docs.